src/components/ContentHighlights/tests/ContentHighlightsDashboard.test.jsx

```diff
@@ -1,89 +1,34 @@
-import { useState } from 'react';
+/* eslint-disable react/prop-types */
```

Comment:
[suggestion] Given that we're disabling this rule across multiple test files, I might recommend we add it to the app's  ESLint config to ignore it wholesale for test files instead of within individual files. See the root `.eslintrc.js` file in the root of the repo for where you should be able to ignore this specific rule for test files.